### PR TITLE
Reconfigure tagging 

### PR DESCRIPTION
**Pull request by uyscuti-wiki**
[T239025](https://phabricator.wikimedia.org/T239025)

### CODE REVIEW

File: TWLight/resources/views.py [hunk 1]
```diff
@@ -45,6 +45,19 @@ def get_queryset(self):
...
+        context = super(PartnersFilterView, self).get_context_data(**kwargs)
+        try:
+            filter_data = kwargs.pop("filter").data
+            tag_id = filter_data.get("tags")
+            if tag_id:
+                context["tag"] = TextFieldTag.objects.get(id=tag_id)
+        except KeyError:
+            pass
+        except TextFieldTag.DoesNotExist:
+            pass
```
Comment (jsnshrmn) [inline TWLight/resources/views.py:58]: could be streamlined to
```
        except (KeyError, TextFieldTag.DoesNotExist):
            pass
```

File: TWLight/resources/tests.py [hunk 2]
```diff
@@ -826,3 +826,33 @@ def test_auto_disable_waitlist_command(self):
         # No change should've been made to the partner with zero accounts available
         self.partner1.refresh_from_db()
         self.assertEqual(self.partner1.status, Partner.WAITLIST)
+
+
+class PartnerTagTest(TestCase):
+    def test_tag_filtering_with_meta_url(self):
```
Comment (jsnshrmn) [inline TWLight/resources/tests.py:832]: please add some comments about what this test is proving.

File: TWLight/resources/views.py [hunk 3]
```diff
@@ -45,6 +45,19 @@ def get_queryset(self):
         else:
             return Partner.objects.order_by("company_name")
 
+    def get_context_data(self, **kwargs):
```
Comment (jsnshrmn) [inline TWLight/resources/views.py:48]: please add comments about what this is doing

**Comment by uyscuti-wiki** [comment]
Believe this too, is ready.

**Pull request merged**